src/ducks/upload/styles.styl

```diff
@@ -135,3 +135,40 @@ progress.upload-queue-progress::-moz-progress-bar
         @extend $icon-16
         @extend $icon
         background-image embedurl('./assets/icons/icon-checked.svg')
+
+@media (max-width: (1023/basefont)rem)
+    // we need to cover the border-bottom of the cozy-bar
+    $coz-bar-size = 48px
+    $coz-bar-border-size = 1px
+
+    .upload-queue
+        position static
+        transform none
+        z-index $app-index
+        max-width 100%
+        height auto
+        max-height 0
+        border-radius 0
+        border 0
+        background 0
+        box-shadow none
+
+        &:before
+            position fixed
+            top $coz-bar-size - $coz-bar-border-size
+            z-index  $nav-index
+            width 100%
+            height $coz-bar-border-size
+            content ""
+            background white
+
+    .upload-queue--visible
+        max-height 30%
```

Comment:
`.upload-queue` is never removed from the DOM; on desktop, it's always there but outside the viewport, and it transitions in when an upload starts. So on mobile, I used a "slide down" sort of animation to make it appear (it has a `max-height` of 0 by default). `30%` is probably way too much now, initially I was also displaying the details of the uploaded files, but that's no longer the case.
Do you see another approach or a better animation for this?